src/stripe-config.js

```diff
@@ -1,19 +1,299 @@
+    },
+  },
+  {
+    // Lithuania
+    code: 'LT',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Luxembourg
+    code: 'LU',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Mexico
+    code: 'MX',
+    currency: 'MXN',
+    accountConfig: {
+      clabe: true,
+    },
+  },
+  {
+    // Netherlands
+    code: 'NL',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // New Zealand
+    code: 'NZ',
+    currency: 'NZD',
+    accountConfig: {
+      accountNumber: true,
+    },
+  },
+  {
+    // Norway
+    code: 'NO',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Poland
+    code: 'PL',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Portugal
+    code: 'PT',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Singapore
+    code: 'SG',
+    currency: 'SGD',
+    accountConfig: {
+      bankCode: true,
+      branchCode: true,
+      accountNumber: true,
+    },
+  },
+  {
+    // Slovakia
+    code: 'SK',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Slovenia
+    code: 'SI',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Spain
+    code: 'ES',
+    currency: 'EUR',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Sweden
+    code: 'SE',
+    currency: 'SEK',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // Switzerland
+    code: 'CH',
+    currency: 'CHF',
+    accountConfig: {
+      iban: true,
+    },
+  },
+  {
+    // United Kingdom
+    code: 'GB',
+    currency: 'GBP',
+    accountConfig: {
+      sortCode: true,
+      accountNumber: true,
+    },
+  },
+  {
+    // United States
+    code: 'US',
+    currency: 'USD',
+    accountConfig: {
+      routingNumber: true,
+      accountNumber: true,
+    },
+  },
+];
+
+/*
+NOTE: This configuration will not be updated!
+
+With Connect Onboarding Stripe will handle collecting most of the information about user. For new setup we only need the list of countries and accountConfig.
+If you want to handle the whole onboarding flow on your on application, you can use the old PayoutDetailsForm as a starting point. That form uses this configuration option.
+You should make sure that the list of countries is up-to-date and that the config contains all the required infomation you need to collect.
+
+Remember to change the import from config.js if you want to use this config! 
+
+This setup is for API version '2019-02-19' and later.
+If you have an older API version in use, you need to update your Stripe API.
+You can check your Stripe API version from Stripe Dashboard -> Developers.
+Stripe only supports payments in certain countries, see full list
+at https://stripe.com/global
+*/
 export const stripeSupportedCountries = [
```

Comment:
Maybe we should comment these out too?

Or alternatively, create some kind of merge:
```js
export const stripeCountryDetails = useLegacy
  ? stripeCountryDetails.map(c => {
      const legacyConfig = stripeSupportedCountries.find(legacy => legacy.code === c.code);
      const accountConfigMaybe = legacyConfig && legacyConfig.accountConfig ? { accountConfig: legacyConfig.accountConfig } : {};
      return { ...c, ...accountConfigMaybe, /*etc.*/ };
    })
  : stripeCountryDetails;
```